URBNSwiftAlert/Classes/AlertStyler.swift

```diff
@@ -176,6 +176,11 @@ extension AlertStyler {
          */
         public var backgroundColor = UIColor.gray
         
+        /**
+         * Background color of the button container
+        */
+        public var buttonContainerBackgroundColor = UIColor.lightGray
```

Comment:
so, you‘re faking the lines by setting the background color of the container? I like the approach but the API is a bit weird. I’d like to get more input here. @ryang1428 ?

Reply:
Well... I think of it more as using the stack view separator as the lines. It would get pretty tricky to have a centered vertical line between the 2 buttons if you were drawing it using borders

Reply:
I guess this is fine for now. I’m having a hard time coming up with a better API off the top of my head.